src/pipecat/services/lmnt/tts.py

```diff
@@ -135,6 +149,7 @@ async def _connect_websocket(self):
                 "format": self._settings["format"],
                 "sample_rate": self.sample_rate,
                 "language": self._settings["language"],
+                "model": self._model_name,
```

Comment:
This should be `model_name`, which is set in the AIServices base class.
```suggestion
                "model": self.model_name,
```

Reply:
done, thank you!